src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,25 @@
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { PreparePost } from '../../types/PreparePost';
+
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: PreparePost[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <>
+    {posts.map((post => (
+      <PostInfo
+        key={post.id}
+        post={post}
+        user={post.user || {
+          id: 0,
+          name: 'unfound',
+          username: 'unfound',
+          email: 'unfound',
+        }}
+        comments={post.comments}
+      />
```

Comment:
no need to provide `user` and `comments` here, you will do it in PostInfo component
just you provide `post` here and next you will work with it in PostInfo